pkg/daemon/ceph/agent/flexvolume/server.go

```diff
@@ -98,7 +101,7 @@ func (s *FlexvolumeServer) Start(driverVendor, driverName string) error {
 	}
 	s.listeners[unixSocketFile] = listener
 
-	if err := os.Chmod(unixSocketFile, 0770); err != nil {
+	if err := os.Chmod(unixSocketFile, 0600); err != nil {
```

Comment:
@nizamial09 The integration tests are failing consistently. I suspect permission changes like these are causing the issue. Have you been able to test locally that you can create an rbd client? For example, can you create the mysql pod in [this doc](https://rook.io/docs/rook/v1.2/ceph-block.html) and see the volume mounted?